### ISSUE TITLE
Add Photos for Each School

### ISSUE THREAD

**Issue by alexlemann** [codefordurham/school-navigator#97]
Add photos to this Google Drive Folder:
https://drive.google.com/folderview?id=0BzzdkaFnyRitcGhBQ0tmdi0yenc&usp=sharing
Document source on this Checklist:
https://docs.google.com/spreadsheets/d/1h31-Cmx6YfXFc6VxhkUoHRr9qjpvGpsM22NhttgpmoE/edit?usp=sharing
Sample Image: https://durhamschoolnavigator.org/#/school/247/
@victor-rocha can you add instructions for ideal photo size?

**Comment by sarahsba** [comment]
3/4 complete on the mission statement and school hours.  Need to test photos to see how that works.

**Comment by sarahsba** [comment]
finished adding mission statements and school hours.  I tried to upload a few photos, but I'm not sure this is worth the effort.  Take a look at the ones I uploaded.  Some of the files were too large and others weren't downloadable from the website so even if we want to include photos, we won't have one every for school.

**Comment by jaycee** [comment]
Links to achiol

**Comment by jaycee** [comment]
Sorry, fat fingered my phone.

We have links to schools already, which I think is more useful than adding pictures. Pictures seem to always end up out of date with a quickness in situations where they need to be manually updated.

**Comment by lemanal** [comment]
Fine with pushing thinking about photos to the backlog. Thanks @sarahsba

**Comment by nickpassarella** [comment]
Hey all. Apologies for not having been at meetups lately. This is short notice, but tomorrow I might have an opportunity to test out a new product we are working on for automating data focused content generation. I saw mission statements have been added, but I could potentially use my testing time to generate descriptions for each school based on specific data.

If this is helpful for this project, let me know what stats should be touched on in the description and I will see what I can do.
